protocols.md

```diff
@@ -0,0 +1,73 @@
+# Protocols
+
+> Protocols are the "attitudes" or "the manner in which one approaches each and
+every element in our space." They are different than pure rules or
+instructions; they are ways that we establish order and maintain practices
+across our group [...] -- They define the way we ought to proceed or behave in
+different situations. As such, they are normative, or premised on values,
+morals, and an idea of how things ought to be done. They are a manifestation of
+our values.
+>
+> -- [Lab Book, Civic Laboratory for Environmental Action Research (CLEAR)][clear-lab-book]
+
+## Scheduling a meeting
+
+Note: Consider whether this meeting might be a global interest to other
+  members. Skip below steps at your discretion for low-stakes topics.
+
+- Try to announce your intention to schedule a meeting on a topic, and ask
+  who's interested to participate. Allow a day for people to respond.
```

Comment:
```suggestion
  who's interested to participate. Allow _at least_ a day for people to respond.
```

I'd say if our internal turnaround time is 48hours 1 day is a short minimum

Reply:
```suggestion
  who's interested to participate. Allow _at least_ 48 hours for people to respond.
```

Your commit suggestion didn't align with my interpretation of your concern. Should I accept mine or yours? :)

Reply:
(Also, in general, should we air toward hours or days with this stuff -- specificity or intentional generality?)

Reply:
go for yours. don't think days/hours matters with this given our specificity elsewhere